python/lsst/ts/m2gui/utils.py

```diff
@@ -111,3 +111,25 @@ def create_label(name="", point_size=None, is_bold=False):
     label.setFont(font)
 
     return label
+
+
+def create_group_box(name, layout):
+    """Create the group box.
+
+    Parameters
+    ----------
+    name : `str`, optional
```

Comment:
Name isn't optional

Reply:
Fixed in the commit: [ba3dc99](https://github.com/lsst-ts/ts_m2gui/pull/6/commits/ba3dc99a36917fc7c61fb23234f4562b38714ea6). Thanks!